tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_device_RTL.py

```diff
@@ -22,11 +22,33 @@ def test_settings_app(self):
 
         moreinfo_page = device_info_page.tap_more_info()
         self.take_screenshot('dev_info-more_info')
-        GaiaImageCompareTestCase.scroll(self.marionette, 'down', device_info_page.screen_element.size['height'],
-                                        screen = moreinfo_page.screen)
+        GaiaImageCompareTestCase.scroll(self.marionette, 'down', moreinfo_page.screen_element.size['height'],
+                                        screen=moreinfo_page.screen_element)
         self.take_screenshot('dev_info-more_info')
         settings.return_to_prev_menu(device_info_page.screen_element)
 
+        device_info_page.tap_your_rights()
+        self.take_screenshot('dev_info-rights')
+        GaiaImageCompareTestCase.scroll(self.marionette, 'down', device_info_page.rights_screen_element.size['height'],
+                                        screen = device_info_page.rights_screen_element)
```

Comment:
Instead of .size, you need to use .rect, I believe, see: http://mxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#122